### PR TITLE
Overview Hardware Devices

### PR DESCRIPTION
**Pull request by upalatucci**
## 🎥 Demo

Screenshot from 2022-04-05 15-51-17
Screenshot from 2022-04-05 15-51-37
Screenshot from 2022-04-05 15-54-16

With editing not enabled
Screenshot from 2022-04-05 16-56-00

### CODE REVIEW

File: src/views/catalog/wizard/tabs/overview/WizardOverviewTab.tsx [hunk 1]
```diff
@@ -31,6 +34,15 @@ const WizardOverviewTab: React.FC<WizardVMContextType> = ({ vm, tabsData, update
   const disks = vm?.spec?.template?.spec?.domain?.devices?.disks;
   const displayName = tabsData?.overview?.templateMetadata?.displayName;
 
+  const hostDevices = getHostDevices(vm) || [];
+  const gpus = getGPUDevices(vm) || [];
+  const nDevices = hostDevices.length + gpus.length;
+
+  const onHardwareDevicesUpdate = React.useCallback(
```
[the remark this answers is not in the document]
Comment (upalatucci) [reply src/views/catalog/wizard/tabs/overview/WizardOverviewTab.tsx:41]: ah yes :smile:

File: src/views/catalog/utils/WizardVMContext/utils/vm-produce.ts [hunk 2]
```diff
@@ -51,3 +51,20 @@ export const produceVMDisks = (
     updateDisks(draftVM);
   });
 };
+
+export const produceVMDevices = (
+  vm: V1VirtualMachine,
+  updateDisks: (vmDraft: WritableDraft<V1VirtualMachine>) => void,
```
Comment (glekner) [inline src/views/catalog/utils/WizardVMContext/utils/vm-produce.ts:57]: change to `updateDevices`

**Comment by avivtur** [comment]
/lgtm
good work :1st_place_medal:

**Pull request merged**